src/confcom/azext_confcom/security_policy.py

```diff
@@ -490,6 +514,7 @@ def pull_image(self, image: ContainerImage) -> Any:
         return client.images.pull(image.base, image.tag)
 
 
+# pylint: disable=R0914,
```

Comment:
maybe add a comment as to why this is needed so it doesn't get deleted by accident later